packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/perl/changeArg.yml

```diff
@@ -0,0 +1,22 @@
+languageId: perl
```

Comment:
Worth adding a couple "chuck arg" tests as well to make sure it cleans up the comma

Reply:
Added those tests (which revealed the behaviour needed improving, which is done).